aepp/src/views/desktop/Admin.vue

```diff
@@ -157,15 +157,15 @@
 </template>
 
 <script>
-  import { Universal, Node } from '@aeternity/aepp-sdk/es'
+  import { AeSdk, Node } from '@aeternity/aepp-sdk'
   import Util from '~/utils/blockchainUtil'
   import { AeBadge, AeButton, AeIcon, AeLoader } from '@aeternity/aepp-components/src/components'
   import BiggerLoader from '~/components/BiggerLoader'
   import BigNumber from 'bignumber.js'
   import config from '~/config'
   import axios from 'axios'
-  import contractSource from '~/assets/GraffitiAuction.aes'
-  import aeternity from '../../utils/aeternityNetwork'
+  import GraffitiAuctionACI from '../../utils/GraffitiAuctionACI'
+  import {nodes} from '../../utils/aeternityNetwork'
```

Comment:
```suggestion
  import { nodes } from '../../utils/aeternityNetwork'
```

Reply:
formatting is not very consistent anyways, maybe we can introduce prettier in the future